src/compare.py

```diff
@@ -1,12 +1,24 @@
 from math import sqrt
 
 import cv2
+import Levenshtein
 import numpy as np
+
+# TODO: easyocr vs. pytesseract?
+# tesseract seems to cause less overall CPU load
+# from easyocr import Reader
```

Comment:
Performance vs ease of use is a valid concern, and unless a method has other outstanding issues, I don't see why we can't offer both to users.
You don't have to do it in this PR, but if you do: Simply add a "Default OCR Method" under "Default Comparison Method" and it can re-use the same "comparison method index" (`^`) for image-specific settings.